unblob/processing.py

```diff
@@ -375,6 +375,13 @@ def _extract_chunk(self, file, chunk: ValidChunk):
             extract_dir = self.carve_dir / (inpath.name + self.config.extract_suffix)
             carved_path = inpath
 
+        # We only extract every blob once, it's a mistake to extract the same blob again
```

Comment:
the comment is miss-leading as there could be other reason why the directory exists:
- for some reason we extract the blob
- the content actually contains a directory with the same name as we would extract, in this case we would also skip extraction.

Unfortunately we do not know which is the above case, but we can report the error

Reply:
This was copied over, but was a good point.

It was a comment that made me re-think if this is the only place we need this check, and eventually led to more work.